src/app/components/videochat/jh-video-chat.html

```diff
@@ -1,65 +1,137 @@
-<div class="row">
-  <header>
-    <div class="row">
-      <div class="hidden-xs col-sm-2 text-center">
-        <jh-footer></jh-footer>
-      </div>
-      <div class="col-xs-6 col-sm-5">
-        <jh-room-info></jh-room-info>
+<div id="wrapper">
+  <!-- Sidebar -->
+  <div id="sidebar-wrapper">
+    <ul class="sidebar-nav">
+      <!-- Info Button -->
+      <div class="sidebar-box" id="sidebar-box-info">
         <button
-           class="btn"
+           class="btn btn-sidebar"
+           id="btn-info"
            ng-click="vm.showHotkeys()"
            title="Keyboard shortcuts">
-          <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+           <span><img class="btn-img" src="images/buttons/buttons_info.svg"/></span>
         </button>
-        <jh-thumbnails-mode-button></jh-thumbnails-mode-button>
-        <jh-leave-button></jh-leave-button>
       </div>
-      <div class="col-xs-6 col-sm-5 text-right">
-        <jh-main-feed-info></jh-main-feed-info>
-        <jh-screen-share-button></jh-screen-share-button>
-        <jh-video-button feed="vm.mainFeed()"></jh-video-button>
+      <!-- Layout Change Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_layout-change.svg"/></span>
+        </button>
+      </div>
+      <!-- Settings Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_settings.svg"/></span>
+        </button>
+      </div>
+      <!-- Hotkey Button -->
+      <div class="sidebar-box">
         <jh-pushtotalk-button></jh-pushtotalk-button>
+      </div>
+      <!-- Audio Button -->
+      <div class="sidebar-box">
         <jh-audio-button feed="vm.mainFeed()"></jh-audio-button>
       </div>
-    </div>
-  </header>
-</div>
+      <!-- Video Button -->
+      <div class="sidebar-box">
+        <jh-video-button feed="vm.mainFeed()"></jh-video-button>
+      </div>
+      <!-- Window Share Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_window-share.svg"/></span>
+        </button>
+      </div>
+      <!-- Screen Share Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_screen-share.svg"/></span>
```

Comment:
the ```<span>``` is not needed to hold the ```<img>```, you can remove it from all the other buttons :)